docs/integration-test-helpers/masonry/MasonryContainer.js

```diff
@@ -86,33 +86,45 @@ export default class MasonryContainer extends Component<Props<{ ... }>, State> {
 
   randomNumberSeed: number = 0;
 
-  componentDidMount() {
-    window.TEST_FETCH_COUNTS = 0;
+  triggerReflow: () => void = () => {
+    if (this.gridRef.current) {
+      this.gridRef.current.reflow();
+      this.forceUpdate();
+    }
+  };
 
-    window.addEventListener('trigger-reflow', () => {
-      if (this.gridRef.current) {
-        this.gridRef.current.reflow();
-        this.forceUpdate();
-      }
+  // $FlowFixMe[unclear-type]
+  setMasonryItems: (e: { detail: { items: $ReadOnlyArray<Object> } }) => void = (e) => {
+    this.setState({
+      items: e.detail.items,
     });
+  };
 
-    window.addEventListener('set-masonry-items', (e) => {
-      this.setState({
-        items: e.detail.items,
-      });
-    });
+  // eslint-disable-next-line class-methods-use-this
+  countError: () => void = () => {
+    window.ERROR_COUNT += 1;
+  };
```

Comment:
just curious, why do we need to count errors here?

Reply:
Not completely sure but I see that we have two integration test on playwright that check for that variable. `playwright/masonry/null-items.spec.mjs` and `playwright/masonry/handle-update-during-insertions.spec.mjs`.

Reply:
ah i see thanks. well PR LGTM but i cant approve it.